backend/gradlew.bat

```diff
@@ -14,7 +14,7 @@
 @rem limitations under the License.

```

Comment:
this file is bundled with the spring release, obtained from here https://start.spring.io/